ci/jenkins/test-vmc.sh

```diff
@@ -568,6 +570,8 @@ function run_conformance {
     ${SSH_WITH_ANTREA_CI_KEY} -n capv@${control_plane_ip} "if [ ! -d ".kube" ]; then mkdir .kube; fi"
     ${SCP_WITH_ANTREA_CI_KEY} $GIT_CHECKOUT_DIR/jenkins/out/kubeconfig capv@${control_plane_ip}:~/.kube/config
 
+    kubectl taint nodes -l node-role.kubernetes.io/control-plane!=true node.cluster.x-k8s.io/uninitialized-
```

Comment:
I suppose we should ignore error here with 'set -e' and 'set +e'?

Reply:
Yes, I found the error can block the test, updated.